File: week04-2.py
#week04-2.py是今天Leecode挑戰題 Easy題(二合一)
#Leecode 2529. Maximum Count of Positive Integar and Negative Integar
#找出有幾個整數，有幾個負數，大的那個數量回傳
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos=0
        neg=0
        for i in range(len(nums)):
            if nums[i] > 0: pos +=1
            if nums[i] < 0: neg +=1

        return max(pos,neg)N

File: week04-2.c
//week04-2.c是今天Leecode挑戰題 Easy題(二合一)
// Leecode 2529. Maximum Count of Positive Integar and Negative Integar
int maximumCount(int* nums, int numsSize) {

    int pos=0,neg=0;
    for(int i=0; i<numsSize; i++){
        if(nums[i]> 0 )pos++;//正數
        if(nums[i]< 0 )neg++;//負數
    }
    if(pos>neg) return pos;
    else return neg;
}
